src/components/Output/custom-els/RangeInput/index.ts

```diff
@@ -0,0 +1,94 @@
+import './styles.css';
+
+const EVENTS = [
+  'input',
+  'change',
+  'focus',
+  'blur',
+];
+
+const PROPERTIES = [
+  'name',
+  'value',
+  'min',
+  'max',
+  'step',
+];
+
+interface RangeInputElement {
+  value: string;
+  min?: string;
+  max?: string;
+  step?: string;
+  precision?: number | string;
+}
+
+class RangeInputElement extends HTMLElement {
+  private _input = document.createElement('input');
+  private _valueDisplayWrapper = document.createElement('aside');
```

Comment:
This doesn't feel like something a screen reader should be interested in, so just go with div/span and aria-hidden. The value is just noise compared to what the input already provides.

Reply:
tis now a div